mail/tasks.py

```
@@ -211,6 +211,10 @@ def send_licence_data_to_hmrc():
                 return
 
             mail = build_licence_data_mail(licences)
+            if mail is None:
+                logging.error("Mail not created, errors in edifact file")
+                return
```

Same as the earlier comment. 

It is more Pythonic to raise and catch specific exceptions instead of relying on the return type to be `None`.

---

agree, updating this.